src/revivalpmmp/pureentities/event/ExplosionPrimeEvent.php

```diff
@@ -1,31 +1,16 @@
 <?php
 
-/*  PureEntitiesX: Mob AI Plugin for PMMP
-    Copyright (C) 2017 RevivalPMMP
-
-    This program is free software: you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation, either version 3 of the License, or
-    any later version.
-
-    This program is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License
-    along with this program.  If not, see <http://www.gnu.org/licenses/>. */
```

Comment:
Not by intend ... sorry. I will re-add copyright to all classes when reformatting all classes when merge is done.